linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_backup_etc_group/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+title: 'Verify Group Who Owns Backup group File'
+
+description: '{{{ describe_file_group_owner(file="/etc/group-", group="root") }}}'
+
+rationale: |-
+    The <tt>/etc/group-</tt> file is a backup file of the <tt>/etc/group</tt>, and as such
```

Comment:
nitpick: `backup file of <tt>/etc/group</tt>, and as such, it also contains`

Reply:
@yuumasato missed the comma `,` after the words `and as such` in each of these.

Reply:
I should be correct now.